### DISCUSSION TITLE
Virtualisation

### DISCUSSION THREAD

**Discussion by paulm17** [icflorescu/mantine-datatable#25] [Ideas]
Taken from the Discord channel:

> Is this virtualization built-in?

> I also thought about opening an issue for virtualization but I think that it's out of scope for that project

My 2c, take a look at https://virtuoso.dev/hello-table/.  Virtuoso is one of the easiest virtualization components out there to use.  👍

**Comment by icflorescu** [comment]
Thanks, will have a look.

**Comment by jnachtigall** [comment]
Related / duplicate of #311

**Reply by paulm17** [reply]
Mine: Sep 10, 2022
Theirs: on Feb 6, 2023

Just saying.  😁
